data/moves.js

```diff
@@ -5214,6 +5214,17 @@ exports.BattleMovedex = {
 				}
 			}
 		},
+		onAfterSubDamage: function (target, source) {
+			let allyActive = target.side.active;
+			if (allyActive.length === 1) {
+				return;
+			}
+			for (let i = 0; i < allyActive.length; i++) {
```

Comment:
We use `for...of` now.

Reply:
... I guessed you wanted me to fix the code I copied from too...